homeassistant/components/kostal_plenticore/sensor.py

```diff
@@ -33,6 +35,23 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+
+@dataclass
+class PlenticoreRequiredKeysMixin:
+    """A class that describes required properties for plenticore sensor entities."""
+
+    module_id: str
+    properties: dict[str, Any]
+    formatter: str
+
+
+@dataclass
+class PlenticoreSensorEntityDescription(
+    SensorEntityDescription, PlenticoreRequiredKeysMixin
+):
```

Comment:
In this PR, I am only converting the tuples to `SensorEntityDescription`.
The old `process data id` is mapped to `key`, and `entity name suffix` is mapped to `name` in the standard `EntityDescription`.

Passing the `EntityDescription` to `PlenticoreDataSensor` will be done in a follow-up PR.